airbyte-integrations/connectors/source-hubspot/acceptance-test-config.yml

```diff
@@ -5,8 +5,8 @@ tests:
   connection:
     - config_path: "secrets/config.json"
       status: "succeed"
-    - config_path: "integration_tests/invalid_config.json"
-      status: "exception"
```

Comment:
why was this removed? we should put it back

Reply:
It's failing, and the best way to fix it I think is to migrate this connector to the CDK, here's the issue https://github.com/airbytehq/airbyte/issues/5114

Reply:
why is it failing?

Reply:
```

self = <source_acceptance_test.tests.test_core.TestConnection object at 0x7fefd1176b90>
connector_config = SecretDict(******)
inputs = ConnectionTestConfig(config_path='integration_tests/invalid_config.json', status=<Status.Exception: 'exception'>, timeout_seconds=None)
docker_runner = <source_acceptance_test.utils.connector_runner.ConnectorRunner object at 0x7fefd1176210>

    def test_check(self, connector_config, inputs: ConnectionTestConfig, docker_runner: ConnectorRunner):
 
    ...
    
            assert err.value.exit_status != 0, "Connector should exit with error code"
>           assert "Traceback" in err.value.stderr.decode("utf-8"), "Connector should print exception"
E           AttributeError: 'str' object has no attribute 'decode'

/usr/local/lib/python3.7/site-packages/source_acceptance_test/tests/test_core.py:92: AttributeError
=========================== short test summary info ============================
FAILED test_core.py::TestConnection::test_check[inputs1] - AttributeError: 's...
============== 1 failed, 8 passed, 2 skipped in 291.69s (0:04:51) ==============
```

Reply:
should this be just "failed" then? See https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py#L74

Reply:
You're right, changed, works now, thank you!